content/channels/index.textile

```diff
@@ -447,6 +447,72 @@ To publish a single message to multiple channels, make multiple "@publish()@":/a
 <p>A client can choose whether or not to receive messages that they publish themselves using the "@echoMessages@":/api/realtime-sdk/types#client-options property. This is set in @ClientOptions@ with a default value of @true@. Setting @echoMessages@ to @false@ will stop clients receiving the messages they publish themselves. This property is only available using the realtime interface, as it isn't possible to subscribe to messages using the REST interface.</p>
 </aside>
 
+h3(#server-side-batching). Server-side batching [beta]
+
+Server-side batching optimizes message processing and delivery by grouping multiple messages into batches before sending them to subscribers. This feature reduces the overall message count, cutting costs and mitigating the risk of hitting rate limits during high-throughput scenarios.
+
+h4. Key benefits
+
+* Cost efficiency.
+* Reduced system load.
+* Message rate limit mitigation.
+
+h4. Use cases
+
+The are the primary use cases where server-side batching provides significant value:
+
+* *High-volume chat applications:*
+A live event chat platform experiences large spikes in message volumes during peak moments, such as live sports games or concerts. Managing high-frequency chat messages and reactions from thousands of users simultaneously can increase costs and the risk of hitting message rate limits.
+_*Solution:*_ Implementing server-side batching allows the platform to group multiple chat messages and reactions into single batches before delivering them to subscribers. This reduces the number of individual messages processed and helps maintain a seamless and interactive user experience during high-activity periods.
+
+* *Fan engagement platforms:*
+A sports fan engagement app allows users to send reactions, such as likes and comments, during live matches. During key moments, like a goal being scored, the app faces a surge in user interactions, leading to a spike in message volume.
+_*Solution:*_ Server-side batching can combine multiple reactions into a batch before sending them to subscribers. This helps manage the high volume of interactions efficiently, preventing rate limits from being exceeded and ensuring all reactions are delivered promptly.
+
+* *Cursor tracking:*
+A collaborative workspace application tracks user presence and cursor movements in realtime, showing the live cursor positions of multiple users. Continuous updates on cursor positions from numerous users can generate a high volume of messages, increasing costs and potentially hitting rate limits.
+_*Solution:*_ The application can batch these frequent updates into single messages by applying server-side batching to the cursor and presence messages. This reduces the number of messages processed and sent, lowering costs and maintaining realtime tracking accuracy.
```

Comment:
I'm not sure if we should include cursors before the work is done to create a policy for them.

Reply:
I have removed "cursor tracking"